### PR TITLE
pnmpsnr: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/pnmpsnr.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pnmpsnr
+
+> Calcula la diferencia entre dos imágenes.
```
Comment (tricantivu) [inline pages.es/common/pnmpsnr.md:3]:
```suggestion
> Calcula la diferencia entre dos imagenes.
```

File: pages.es/common/pnmpsnr.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> Más información: <https://netpbm.sourceforge.net/doc/pnmpsnr.html>.
+
+- Calcula la diferencia, es decir, la relación señal-ruido (PSNR) entre dos imágenes:
```
Comment (tricantivu) [inline pages.es/common/pnmpsnr.md:6]:
```suggestion
- Calcula la diferencia, es decir, la relación señal-ruido (PSNR) entre dos imagenes:
```

File: pages.es/common/pnmpsnr.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`pnmpsnr {{ruta/al/archivo1.pnm}} {{ruta/al/archivo2.pnm}}`
+
+- Compara los componentes de color en lugar de los componentes de luminancia y crominancia de las imágenes:
```
Comment (tricantivu) [inline pages.es/common/pnmpsnr.md:10]:
```suggestion
- Compara los componentes de color en lugar de los componentes de luminancia y crominancia de las imagenes:
```

File: pages.es/common/pnmpsnr.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`pnmpsnr {{ruta/al/archivo1.pnm}} {{ruta/al/archivo2.pnm}} -rgb`
+
+- Ejecuta en modo de comparación, es decir, sólo la salida `nomatch` o `match` dependiendo de si el cálculo PSNR supera `n` o no:
```
Comment (tricantivu) [inline pages.es/common/pnmpsnr.md:14]:
```suggestion
- Ejecuta en modo de comparación, es decir, solo la salida `nomatch` o `match` dependiendo de si el cálculo PSNR supera `n` o no:
```

File: pages.es/common/pnmpsnr.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ejecuta en modo comparación y compara los componentes individuales de la imagen, es decir, Y, Cb y Cr, con los umbrales correspondientes:
+
+`pnmpsnr {{ruta/al/archivo1.pnm}} {{ruta/al/archivo2.pnm}} -target1 {{umbral_Y}} -target2 {{umbral_Cb}} -target3 {{umbral_Cr}}`
+
+- Ejecuta en modo comparación y compara los componentes individuales de la imagen, es decir, rojo, verde y azul con los umbrales correspondientes:
+
+`pnmpsnr {{ruta/al/archivo1.pnm}} {{ruta/al/archivo2.pnm}} -rgb -target1 {{umbral_rojo}} -target2 {{umbral_verde}} -target3 {{umbral_azul}}`
+
+- Produce salida legible por ordenadores:
```
Comment (tricantivu) [inline pages.es/common/pnmpsnr.md:26]:
```suggestion
- Produce salida legible para máquinas:
``

File: pages.es/common/pnmpsnr.md (hunk 1 above)
Comment (kant) [reply pages.es/common/pnmpsnr.md:3]: source: https://llevatilde.es/palabra/im%C3%A1genes

File: pages.es/common/pnmpsnr.md (hunk 2 above)
Comment (kant) [reply pages.es/common/pnmpsnr.md:6]: source: https://llevatilde.es/palabra/im%C3%A1genes

File: pages.es/common/pnmpsnr.md (hunk 3 above)
Comment (kant) [reply pages.es/common/pnmpsnr.md:10]: source: https://llevatilde.es/palabra/im%C3%A1genes

File: pages.es/common/pnmpsnr.md (hunk 2 above)
Comment (tricantivu) [reply pages.es/common/pnmpsnr.md:6]: Apparently, the plural form of the word must have an accent but the singular one must not.

> Las voces imagen, dictamen, examen, resumen, volumen, origen, joven, abdomen o margen se escriben sin tilde en singular, pero con tilde en plural.

[Source](https://www.fundeu.es/recomendacion/joven-orden-o-imagen-sin-tilde-en-singular)

I propose ignoring the suggestion of plurals and keeping the singular ones, do you agree?

Comment (kant) [reply pages.es/common/pnmpsnr.md:6]: `Imágenes` is plural, not singular, I did respect the original file: `images` that is not singular.

**Review by tricantivu — APPROVED** [review]
Remaining suggestions are unnecessary, LGTM.

The CI error was my fault, I accidentally introduced the backticks.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**